build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -239,12 +239,7 @@ class RuntimeTestConfig {
       this.browserify.transform = [
         [
           'babelify',
-          {
-            // Transform "node_modules/", but ignore devDependencies.
-            'global': true,
-            'ignore': devDependencies(),
-            'plugins': [plugin],
-          },
+          Object.assign({}, BABELIFY_GLOBAL_TRANSFORM, {plugins: [plugin]}),
```

Comment:
Should we be removing `sourceMapsAbsolute: true` set in karma.conf.js?

Reply:
Good catch. Fixed ina76a1dc8a23c62ba739aaedad114541c521ccd67